back/src/infra/repository/database/IndicatorRepositoryDatabase.ts

```diff
@@ -0,0 +1,60 @@
+import { Indicator } from '@/domain/entity'
+import { IndicatorRepository } from '@/domain/repository'
+import { Connection } from '@/infra/database'
+
+export class IndicatorRepositoryDatabase implements IndicatorRepository {
+  constructor (readonly connection: Connection) {}
+
+  async remove (idIndicator: number): Promise<void> {
+    await this.connection.query('DELETE FROM indicators WHERE id = $1', [idIndicator])
+  }
+
+  async save (indicator: Indicator): Promise<void> {
+    await this.connection.query(`INSERT INTO indicators
+    (
+      name,
+      unit,
+      description,
+      current_value,
+      accepted_value,
+      min_value,
+      max_value
+    ) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING *`, [
+      indicator.name,
+      indicator.unit,
+      indicator.description,
+      indicator.currentValue,
+      indicator.acceptedValue,
+      indicator.minValue,
+      indicator.maxValue
+    ])
+  }
+
+  async list (): Promise<Indicator[]> {
+    const indicatorsData = await this.connection.query('SELECT * FROM indicators', [])
```

Comment:
To not list the creation date
```suggestion
    const indicatorsData = await this.connection.query('SELECT 
      id,
      name,
      unit,
      description,
      current_value,
      accepted_value,
      min_value,
      max_value 
  FROM indicators', [])
```